src/main/com/fulcrologic/fulcro/algorithms/transit_old.cljc

```diff
@@ -0,0 +1,109 @@
+(ns com.fulcrologic.fulcro.algorithms.transit-old
+  "This version is for use with older versions of Transit, such as what is supported on Datomic Cloud
+
+   Transit functions for the on-the-wire EDN communication to common remotes. Includes support for Fulcro tempids,
+   and can be extended to support additional application-specific data types."
+  #?(:clj
+     (:refer-clojure :exclude [ref]))
+  (:require [cognitect.transit :as t]
+            #?(:cljs [com.cognitect.transit :as ct])
+            [com.fulcrologic.fulcro.algorithms.tempid :as tempid #?@(:cljs [:refer [TempId]])])
+  #?(:clj
+     (:import [com.cognitect.transit
+               TransitFactory WriteHandler ReadHandler]
+              [com.fulcrologic.fulcro.algorithms.tempid TempId])))
+
+#?(:cljs
```

Comment:
Is it really necessary to have definitions of these in separate namespaces?  Perhaps it would make sense to centralize these somehow?  Or was it that you would need to move them, which would be an incompatible change?